src/utils/helper.js

```diff
@@ -188,22 +188,32 @@ const uploadFile = ({
     })
 }
 
-const downloadWindowTitle = i18n.t('Loading exported data')
-const downloadWindowHtml = `
-<div style="height: 100%; width: 100%; display: flex; justify-content: center; align-items: center; color: rgb(33, 41, 52)">
-    <p>${downloadWindowTitle}</p>
-</div>
-`
-
 // call stub function if available
-const locationAssign = (url) => {
+const locationAssign = (relativeUrl) => {
```

Comment:
Oh, I thought baseUrl from `useConfig` would be the absolute URL? But it may depend if the it's available from `system/info` I guess?

Very minor: but not sure if the rename of this variable is better when it _can_ be an absolute url as well?